src/go/k8s/config/crd/bases/cluster.redpanda.com_topics.yaml

```diff
@@ -450,6 +450,471 @@ spec:
                         Source is where this config entry is from.
 
 
+                        This field has a default of -1.
+                      type: string
+                    unknownTags:
+                      additionalProperties:
+                        type: string
+                      description: UnknownTags are tags Kafka sent that we do not
+                        know the purpose of.
+                      type: object
+                    value:
+                      description: |-
+                        Value is the value for this config key. If the key is sensitive,
+                        the value will be null.
+                      type: string
+                  required:
+                  - configType
+                  - isDefault
+                  - isSensitive
+                  - name
+                  - readOnly
+                  - source
+                  - unknownTags
+                  type: object
+                type: array
+            type: object
+        type: object
+    served: true
+    storage: false
```

Comment:
I know that v1alpha2 is used as the storage version for the redpanda resource but I'm a bit scarred after the process that got it there (It was the inspiration of the take home you did). Any concerns on your end? I think it's fine but storage changes (especially sans webhooks) always warrant a bit of caution.

Reply:
So I think things should be fine namely because I'm not modifying any of the fields of the underlying CRD. As it is, even with the potential addition of a `ClusterRef` field, that'd apply both the v1alpha1 and v1alpha2 versions, so I'd imagine the two would be pretty much entirely compatible (hence the type definition of `v1alpha1` just being the same as `v1alpha2`).